.teamcity/src/main/kotlin/model/PerformanceTestBucketProvider.kt

```diff
@@ -51,12 +51,10 @@ class StatisticsBasedPerformanceTestBucketProvider(private val model: CIBuildMod
         val scenarios = determineScenariosFor(performanceTestSpec, performanceTestConfigurations)
         val testProjectToScenarioDurations = determineScenarioTestDurations(performanceTestSpec.os, performanceTestDurations)
         val testProjectScenarioDurationsFallback = determineScenarioTestDurations(Os.LINUX, performanceTestDurations)
-        val repetitions = if (performanceTestType == PerformanceTestType.flakinessDetection) 2 else 1
```

Comment:
For some reason I thought we have been running everything 3 times instead of 2 times for flakiness detection. I think 2 times is fine as well.

Reply:
Yeah, that's what I thought as well.